Services/CO.CDP.EntityVerification/appsettings.json

```diff
@@ -5,5 +5,13 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "QueueUrl": "https://localhost.localstack.cloud:4566/000000000000/create-organisation",
```

Comment:
For Development, this should be configured to the localstack service run in Docker.

Reply:
Not sure how else it can be configured, tried to create the queue via aws cli and url is same but prefixed with location:

![image](https://github.com/user-attachments/assets/5aead007-2869-4e07-b415-de70fc1eb44a)

Reply:
Fine then! I thought it's a reference to their cloud service when I looked at it. Localstack must be resolving it somehow 🤷